index.bs

```diff
@@ -4535,7 +4535,12 @@ Note: Algorithms specified in [[!CREDENTIAL-MANAGEMENT-1]] perform the actual pe
 ## Using Web Authentication within <code>iframe</code> elements ## {#sctn-iframe-guidance}
 
 The [=Web Authentication API=] is disabled by default in cross-origin <{iframe}>s.
-To override this default policy and indicate that a cross-origin <{iframe}> is allowed to invoke the [=Web Authentication API=]'s {{PublicKeyCredential/[DISCOVER-METHOD]}} method, specify the <{iframe/allow}> attribute on the <{iframe}> element and include the <code>[=publickey-credentials-get-feature|publickey-credentials-get=]</code> feature-identifier token in the <{iframe/allow}> attribute's value.
+To override this default policy and indicate that a cross-origin <{iframe}> is allowed to invoke the [=Web Authentication API=]'s
```

Comment:
```suggestion
To override this default policy and indicate that a cross-origin <{iframe}> is allowed to invoke the [=Web Authentication API=]s
```

Not possessive.

Reply:
Hm. In that case the phrase becomes "allowed to invoke the Web Authentication APIs [create] and [get] methods". I feel like "methods" becomes hanging there, doesn't it?

But I think that just moving that word would work: "allowed to invoke the Web Authentication API methods [create] and [get]". What do you think?

Reply:
I suppose "allowed to invoke the Web Authentication APIs [create] and [get]" could work too (just delete "methods").

Reply:
You're right.  Just merge ignoring my comment.

Reply:
Ok, thank you!